Sources/LiveViewNative/Utils/UnitPoint.swift

```diff
@@ -6,8 +6,58 @@
 //
 
 import SwiftUI
+import LiveViewNativeCore
+import RegexBuilder
 
-extension UnitPoint: Decodable {
+extension UnitPoint: Decodable, AttributeDecodable {
+    public init(from attribute: LiveViewNativeCore.Attribute?) throws {
+        guard let value = attribute?.value else { throw AttributeDecodingError.missingAttribute(Self.self) }
+        switch value {
+        case "zero":
+            self = .zero
+        case "center":
+            self = .center
+        case "leading":
+            self = .leading
+        case "trailing":
+            self = .trailing
+        case "top":
+            self = .top
+        case "bottom":
+            self = .bottom
+        case "top-leading":
+            self = .topLeading
+        case "top-trailing":
+            self = .topTrailing
+        case "bottom-leading":
+            self = .bottomLeading
+        case "bottom-trailing":
+            self = .bottomTrailing
+        default:
+            let pattern = Regex {
+                Capture {
+                    OneOrMore(.digit)
+                } transform: { Double($0) }
+                OneOrMore {
+                    ChoiceOf {
+                        ","
+                        OneOrMore(.whitespace)
+                    }
+                }
+                Capture {
+                    OneOrMore(.digit)
+                } transform: { Double($0) }
+            }
+            .anchorsMatchLineEndings()
```

Comment:
Does the `.digit` character class properly handle decimals or is it just `[0-9]`?

Reply:
You're right, I'll need to rework that